data/coreteam_members.yml

```
@@ -107,4 +107,9 @@ members:
     description: |
       a computer science undergrad who's having fun learning new things.
       He enjoys reading, football, and foods he hasn't tried before. Reach him at <a href="mailto:jivesh@nushackers.org">jivesh@nushackers.org</a>
+  
+  - name: Azeem
+    description: |
+      a computer science undergrad who loves tinkering and building things
+      He enjoys riding his motorcycle, catching movies and building telegram bots. Reach him at <a href="mailto:azeem@nushackers.org">azeem@nushackers.org</a>
```

Huge UX issues here. I think the user's experience will be completely broken if they click on the link and end up navigating away from the page. This severely affects conversion rates and impressions.

Also, please add a full stop.

Please change to
```suggestion
      He enjoys riding his motorcycle, catching movies and building telegram bots. Reach him at <a href="mailto:azeem@nushackers.org" target="_blank" rel="noopener noreferrer nofollow" >azeem@nushackers.org</a>.
```

---

Fixed